Makefile

```diff
@@ -361,6 +361,11 @@ proto-all: proto-tools proto-gen proto-lint proto-check-breaking proto-swagger-g
 proto-gen:
```

Comment:
Do we still need this target then?

Reply:
I didn't remove it because some people may want to not use docker. I can remove it though. Dont have a preference.

Reply:
i might still use this one.